pkg/bpmn_engine/error.go

```diff
@@ -0,0 +1,5 @@
+package bpmn_engine
+
+import "errors"
+
+var BpmnValidationError = errors.New("bpmn validation error")
```

Comment:
In order to allow varying errors ... let's use a struct

```suggestion
type BpmnValidationError struct {
	Msg string
}

func (e *BpmnValidationError) Error() string {
	return e.Msg
}
```

Reply:
Good idea, I will change the BpmnValidationError  to BpmnEnginerError by the way making the error more widely used in the future.